app/services/bgs/dependent_service.rb

```diff
@@ -2,13 +2,60 @@
 
 module BGS
   class DependentService
-    def get_dependents(current_user)
-      service = BGS::Services.new(
-        external_uid: current_user.icn,
-        external_key: current_user.common_name
+    include SentryLogging
+
+    def initialize(user)
+      @user = user
+    end
+
+    def get_dependents
+      service.claimants.find_dependents_by_participant_id(@user.participant_id, @user.ssn)
+    end
+
+    def submit_686c_form(claim)
+      va_file_number_with_payload = add_va_file_number_to_payload
+
+      VBMS::SubmitDependentsPDFJob.perform_async(claim.id, va_file_number_with_payload)
+    rescue => e
+      report_error(e)
+    end
+
+    private
+
+    def service
+      external_key = @user.common_name || @user.email
+
+      @service ||= BGS::Services.new(
+        external_uid: @user.icn,
+        external_key: external_key
       )
+    end
+
+    def add_va_file_number_to_payload
+      va_file_number = service.people.find_person_by_ptcpnt_id(@user.participant_id)
```

Comment:
What does `service.people` do? Is that also making another service call behind the covers? I'm concerned with someone calling `submit_686c_form` thinking that they're just making one service call, when in reality it could be many more.

Reply:
Also the abstraction reads a bit funky to me. Shouldn't "find person" return a "person" of some kind? e.g. 
```ruby
bgs_person = service.people.find_person_by_ptcpnt_id(@user.participant_id)
```

Reply:
Great question. We are calling `service.people` with the intent to get the user's `va_file_number`, I agree that the naming of the variable is confusing. We were so laser focused on getting the va_file_number that we named it that. I will change the variable name to better convey what that is.

To add some context, once we get that 'lib/bgs' code in that you've been helping us with, we will be passing that va_file_number into both the PDF job and a form submit job. The va_file_number is necessary for both the pdf and the form submission. Our thinking was that it's best to do that call one time to mitigate risk of failure than call it in both jobs.